samples/applets/view-document.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * to run view document applet from the project root folder type in your console:
+ * > node samples/applets/view-document <cliend_id> <client_secret> <username> <password> <path_to_file>
```

Comment:
```suggestion
 * > node samples/applets/view-document <cliend_id> <client_secret> <username> <password> <path_to_file> '<fields_stringified>'
```

Reply:
make `<fields_stringified>` an optional parameter. if it is absent then upload document without fields